filestorage/handler_base.py

```diff
@@ -226,6 +283,82 @@ async def _async_exists(self, item: FileItem) -> bool:
         """Determine if the given filename exists in the storage container."""
         pass
 
+    async def async_size(self, filename: str) -> int:
```

Comment:
I think these should continue to follow the `get_` pattern: `get_size()` and `async_get_size()`. "Size" by itself can be a verb in English - and would be a funky command to implement! ("Make my file this size!" ... truncates file to that size... )

Reply:
I agree, my changes should reflect this new verbiage.